🧪 **Casey** — CI Failing ❌

**`lint-and-test (api)` failed:** TypeScript type error at line 229.

```
error TS2322: Type 'Record<string, unknown>' is not assignable to type 'InputJsonValue | NullableJsonNullValueInput | undefined'.
```

In `form9-export.ts:229` — the `form9Data` field needs to be cast to a Prisma-compatible JSON type. Please fix CI before review.

--- a/api/src/services/form9-export.ts
+++ b/api/src/services/form9-export.ts
@@ -226,7 +226,7 @@ export class Form9ExportService {
     // Also store on the report for caching
     await this.prisma.report.update({
       where: { id: reportId },
-      data: { form9Data: form9Data as unknown as Record<string, unknown> },
+      data: { form9Data: JSON.parse(JSON.stringify(form9Data)) },
     });
 
     return form9Data;

